platform/ui/src/components/FilterDateRange/FilterDateRange.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { DateRange, FilterWrapper } from '@ohif/ui';
+
+const FilterDateRange = ({
+  label,
+  isSortable,
+  isBeingSorted,
+  sortDirection,
+  onLabelClick,
+  inputValue,
+  inputProps,
```

Comment:
This implies a leaky abstraction. Can we just expose the props directly that we want to pass through so the user doesn't have to look up docs for `DateRange`?

Reply:
That would means these would be renamed to `value`. With `inputProps` expanded based on the props we want to pass through.

Reply:
I've updated the code to work only with the required properties on each input component.